.github/workflows/binder-badge.yml

```diff
@@ -0,0 +1,18 @@
+name: binder-badge
```

Comment:
Can you add an inline comment to clarify with a sentence what this workflow does and for what reason?